tests/Http/Middleware/AdminReviewTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Tests\Http\Middleware;
+
+use Tests\TestCase;
+use App\Models\User;
+use App\Models\Movie;
+use App\Models\Review;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+
+class AdminReviewTest extends TestCase
```

Comment:
Hey Jayd, I think that the name here could be changed to AuthUser as we are not really testing if the admin can do reviews. 😄

Reply:
Hi @vrodriguero, sorry I checked all the tests and it passed, would you mind letting me know which ones or send me a screenshot?

Reply:
There are 10 tests failing in mine for the celeb test, that is because the admin middleware was changed. So, when we do a change we have to look at other parts that could be affected. So any test that it might be using the middleware should be checked :D